tests/main/preseed-lxd/task.yaml

```diff
@@ -46,6 +46,7 @@ restore: |
     lxc stop my-ubuntu --force || true
     lxc delete my-ubuntu || true
     apt purge -y lxd
+    apt autoremove --purge -y
```

Comment:
(nitpick) You could do this in a single line: `apt autoremove --purge -y lxd`

Reply:
Oh, I didn't know this. Thanks!